### PR TITLE
New features

### PR DESCRIPTION
**Pull request by S-Dafarra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,5 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$UnicyclePlanner_DIR/lib
 export UnicyclePlanner_DIR=/path/where/you/installed/
 export DYLD_LIBRARY_PATH=$LD_LIBRARY_PATH:$UnicyclePlanner_DIR/lib
 ```
-## Basic Idea
-<img src="https://github.com/robotology/unicycle-footstep-planner/blob/master/images/planner.gif">
-
-## Implementation
-<img src="https://github.com/robotology/unicycle-footstep-planner/blob/master/images/unicycleWalking.gif">
+## Basic Idea & Implementation
+See https://github.com/robotology/unicycle-footstep-planner/issues/7
```
Comment (traversaro) [inline README.md]: Why are not using the old format but using https://user-images.githubusercontent.com/18591940/35742336-68cbf374-083a-11e8-8393-c0a5ea268866.gif and https://user-images.githubusercontent.com/18591940/35742348-71071cb2-083a-11e8-97a7-3577d997df8f.gif ?

File: src/FootPrintsInterpolator.cpp [hunk 2]
```diff
@@ -570,29 +616,113 @@ bool FeetInterpolator::computeLocalZMP(const std::vector<StepPhase> &stepPhase,
...
 }
 
-void FeetInterpolator::computeGlobalZMP()
+iDynTree::Position FeetInterpolator::pos3D(const iDynTree::Vector2 &xy)
+{
+    iDynTree::Position output;
+    output.zero();
+    output(0) = xy(0);
+    output(1) = xy(1);
+    return output;
```
Comment (traversaro) [inline src/FootPrintsInterpolator.cpp]: I personal prefer the compact version `return iDynTree::Position(xy(0), xy(1), 0.0);`, but I would understand the reason for preferring the long version.

File: CMakeLists.txt [hunk 3]
```diff
@@ -4,8 +4,8 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 # Enable C++11
```
Comment (traversaro) [inline CMakeLists.txt]: Now this comment is outdated.

File: CMakeLists.txt [hunk 4]
```diff
@@ -4,8 +4,8 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
...
 set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED 11)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED 14)
```
Comment (traversaro) [inline CMakeLists.txt]: `CMAKE_CXX_STANDARD_REQUIRED` is actually a boolean option, see https://cmake.org/cmake/help/v3.1/variable/CMAKE_CXX_STANDARD_REQUIRED.html . I originally introduced this problem in iDynTree, and it continued to propagate by copy&pasting . : D

File: README.md (hunk 1 above)
Comment (S-Dafarra) [reply README.md]: Those link are automatically generated when loading images in issues. Initially to solve the problem I inserted the images in the repository, and later I recognized it was actually a bad idea. Actually I did not think either to this solution when creating the issue! Thanks

**Pull request merged**